Libraries/DeprecatedPropTypes/DeprecatedImageProps.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+const DeprecatedEdgeInsetsPropType = require('DeprecatedEdgeInsetsPropType');
+const DeprecatedImageSourcePropType = require('DeprecatedImageSourcePropType');
+const DeprecatedImageStylePropTypes = require('DeprecatedImageStylePropTypes');
+const DeprecatedStyleSheetPropType = require('DeprecatedStyleSheetPropType');
+const PropTypes = require('prop-types');
+
+module.exports = {
+  /**
+   * See https://facebook.github.io/react-native/docs/image.html#style
+   */
+  style: DeprecatedStyleSheetPropType(DeprecatedImageStylePropTypes),
```

Comment:
In general, we should have a singular source of truth for the documentation, which should be the flow types. So, could you move these comments to the `ImageProps` flow type?

Reply:
While moving comments over, I noticed a few discrepancies between Flow properties and propType properties. The following are declared as propTypes (in DeprecatedImageProps, soon to be renamed DeprecatedImagePropType) but not in Flow (in ImageProps):
* accessible: PropTypes.bool
* accessibilityLabel: PropTypes.node
* resizeMode: PropTypes.oneOf([
    'cover',
    'contain',
    'stretch',
    'repeat',
    'center',
  ])
* testID: PropTypes.string
* onLayout: PropTypes.func

Are these not needed in the Flow definitions, or are they missing from the Flow definitions?

Reply:
Oh, interesting. We should move these properties to the flow definitions.